web/cobrands/sass/_base.scss

```diff
@@ -2392,6 +2392,39 @@ a#geolocate_link.loading, .btn--geolocate.loading {
     vertical-align: -2px;
 }
 
+$nicetable-hover-background: rgba($primary, 0.5);
+
+.nicetable {
+    width: 100%;
+    margin-bottom: 1.5em;
+
+    td, th {
+        padding: 0.6em 1.5em 0.6em 0;
```

Comment:
Should this be `flip`ed for RTL? And the last-child below.

Reply:
Yes, it should, because `:last-child` will match the _left_-most child in RTL languages. Good catch!